src/core/java/xyz/wagyourtail/jsmacros/core/service/ServiceManager.java

```diff
@@ -382,4 +468,14 @@ public enum ServiceStatus {
         UNKNOWN // service doesn't exist
     }
 
+    static class SecretFields {
+        private static final SecretFields EMPTY = new SecretFields();
```

Comment:
EventService and Service manager are in the same package. so wouldn't package-private fields give the desired result?

Reply:
> EventService and ServiceManager are in the same package.

OH. didn't notice that.